packages/trader/src/AppV2/Components/TradeParameters/LastDigitPrediction/last-digit-selector.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Digit from './digit';
+
+type TLastDigitSelectorProps = {
+    digits: number[];
+    digit_stats: number[];
+    is_stats_mode?: boolean;
+    onDigitSelect?: (digit: number) => void;
+    selected_digit?: number;
+};
+
+const LastDigitSelector = ({
+    digits = [],
+    digit_stats,
+    is_stats_mode,
+    onDigitSelect,
+    selected_digit,
+}: TLastDigitSelectorProps) => (
+    <div className='last-digit-prediction__selector'>
+        {[...Array(2).keys()].map(row_key => (
+            <div key={row_key} className='last-digit-prediction__selector-row'>
+                {digits.slice(row_key * 5, (row_key + 1) * 5).map(digit => (
```

Comment:
whats happening with calculations here

Reply:
@henry-deriv just math: there will be 2 rows filled with 5 `digits` each :)

Reply:
got it